src/ducks/context/SelectionContext.jsx

```diff
@@ -30,7 +30,8 @@ const SelectionProvider = ({ children }) => {
   const [selected, setSelected] = useState([])
   const [isSelectionModeActive, setIsSelectionModeActive] = useState(false)
 
-  const isSelectionModeEnabled = flag('banks.selectionMode.enabled')
+  const isSelectionModeEnabled =
+    flag('banks.selectionMode.enabled') === false ? false : true // to activate the feature if the flag is null or undefined
```

Comment:
Not sure about the spec but we could refactor the statement to `const isSelectionModeEnabled = `flag('banks.selectionMode.enabled')` couldn't we? Can flag() return a non-boolean?

Reply:
oui ça peut retourner `null` c'est pour ça que j'ai mis le commentaire avec

Reply:
Je vois, pas top comme comportement je trouve; dans ce cas je propose `const isSelectionModeEnabled = Boolean(flag('banks.selectionMode.enabled'))` ou `const isSelectionModeEnabled = !!flag('banks.selectionMode.enabled')`

Reply:
>Pour désactiver le fait que la feature soit derrière un flag, et donc pour la rendre accessible à tout le monde.

Pourquoi ne pas retirer le flag tout simplement ? 🤔

Reply:
c'pas faux !